.dockerignore

```diff
@@ -8,6 +8,8 @@ run
 tmp
 log
 
+docs/site
```

Comment:
Shouldn't this be `_site`? I know you copied it from that other `.dockerignore` file but I think this might have been incorrect.

Reply:
Updated